shop4All-backend/src/main/java/com/example/shop4All_backend/controllers/UserController.java

```diff
@@ -3,6 +3,7 @@
 import com.example.shop4All_backend.entities.User;
 import com.example.shop4All_backend.services.UserService;
 import jakarta.annotation.PostConstruct;
+import jakarta.persistence.PreRemove;
```

Comment:
Is this being used somewhere? If not, please delete it and all the unused imports.